### PR TITLE
LibGroup chapter numbers fix

### LINKED ISSUE #3553: MangaLib: Chapter numbers mismatch
### Source information

MangaLib 1.4.102

### Source language

Russian

### Steps to reproduce

1. Open any series with extra chapters or missing chapters (for example Weakling Pedal)

### Here's what it looks like before URL change: source title numbers and chapter numbers are the same

Screenshot_2024-06-12-21-12-54-30_a603e4da04bada03cae438f177815547

Screenshot_2024-06-12-21-14-53-84_a603e4da04bada03cae438f177815547 

Screenshot_2024-06-12-21-14-58-00_a603e4da04bada03cae438f177815547

###  Here's what it looks like after URL change: no missing chapters alert, chapter numbers don't correspond to the actual numbers, tracking services receive incorrect chapter numbers

Screenshot_2024-06-12-21-15-41-08_a603e4da04bada03cae438f177815547

Screenshot_2024-06-12-21-15-44-49_a603e4da04bada03cae438f177815547

### Mihon/Tachiyomi version

TachiyomiSY 1.10.5

### Android version

Android 11

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by romshke**
Closes #3553

Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [ ] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

**Comment by Smol-Ame** [comment]
Write "Closes #3553" in this PR request's description instead, as listed in one of the checklist items

File: lib-multisrc/libgroup/src/eu/kanade/tachiyomi/multisrc/libgroup/LibGroupDto.kt [hunk 1]
```diff
@@ -241,7 +241,7 @@ class Chapter(
         url = "/$slugUrl/chapter?$branchStr&volume=$volume&number=$number"
         scanlator = getTeamName(branchId) ?: if (isScanUser) getUserName(branchId) else null
         date_upload = runCatching { LibGroup.simpleDateFormat.parse(first(branchId)!!.createdAt)!!.time }.getOrDefault(0L)
-        chapter_number = itemNumber ?: -1f
```
Comment (AwkwardPeak7) [inline lib-multisrc/libgroup/src/eu/kanade/tachiyomi/multisrc/libgroup/LibGroupDto.kt:244]: remove from dto too if not using

**Review by AwkwardPeak7 — CHANGES_REQUESTED** [review]

**Review by AwkwardPeak7 — APPROVED** [review]

**Comment by BlueCat300** [comment]
After this update app crashes during global(all sources) search manga.

**Comment by Smol-Ame** [comment]
> After this update app crashes during global(all sources) search manga.

@BlueCat300 could you come to the Keiyoushi Discord repo-support channel to provide some extra info? At least on my end with a fresh extension install, it doesn't seem to crash

**Pull request merged**